### ISSUE TITLE
First launch of 4.6.0 version fires a fatal error "Call to undefined function PMA\libraries\PMA_langSet()"

### ISSUE THREAD

**Issue by cFreed-AKA-Alf** [phpmyadmin/phpmyadmin#12210]
### Steps to reproduce

1. **Not sure this precise situation is needed to encounter the issue (see Investigation below)**
Being in the context of EasyPHP DevServer 14.IVC11, with PMA 4.1.4, update to 4.6.0 (following the guide found at http://forum.codeigniter-france.com/threads/352-Maj-de-PhpMyAdmin-dans-EasyPhp), so the 2 versions (4.1.4 and 4.6.0) are both available.

2. **Again, not sure this precise situation is needed to encounter the issue (see Investigation below)**
Having a 4.1.4 instance open, launch 4.6.0 in a new tab. Then it fires this error:
> Fatal error: Call to undefined function PMA\libraries\PMA_langSet() in C:\Program Files (x86)\EasyPHP-DevServer-14.1VC11\modules\phpMyAdmin-4.6.0-all-languages\libraries\Config.php on line 1021

3. I found the exactly same error reported in a different context: http://stackoverflow.com/questions/36663228/resolved-upgrading-phpmyadmin-in-xampp-to-latest-produces-an-error.
The solution proposed (refresh browser's cache) didn't apply for me, since I always work with my browser's cache deactivated.  
The error was also reported at http://stackoverflow.com/questions/36352495/fatal-error-cannot-redeclare-previously-declared-in-d-wamp-apps-phpmyadm. I tried the proposed solution (delete the session cookies) without success: the cookies were simply recreated by the next try, firing the same error.  
Finally I got rid of the problem after merely killing the tab containing the 4.1.4 instance.
I can't at all figure out how it would be related to the issue.

### Investigation

Anyway, even before finding how to escape from the issue, I tried to understand what happened.
**Then I looked for the `PMA_langset()` function and discovered that it doesn't exist anywhere in the 4.6.0 version!** (while in 4.1.4 it was located in `libraries/select_lang.lib.php`, which doesn't exist in 4.6.0).

So I tend to think that it might be a bug: the line 1021 shouldn't appear any longer.
And it seems that the issue happens (in various contexts) depending on a combination of factors that make the program to reach this branch of the lines 1011-1028, at the time of the first launch.
After that (till now!) I didn't experience the issue any more, even after killing the tab and re-launching PMA.

Hope this helps.

### Server configuration
**Operating system**: Windows 7

**Web server:** Apache 2.4

**Database:** N/A

**PHP version:** 5.6.15

**phpMyAdmin version:** 4.6.0

### Client configuration
**Browser:** Firefox 45.0.2

**Operating system:** Windows 7

**Comment by ibennetch** [comment]
Thanks for the detailed research.

I believe this section of code has been improved with https://github.com/phpmyadmin/phpmyadmin/commit/999db1491c86bd669b1a8aad1c630159b577551b 

Could you try the latest 'master' branch (if you don't have git handy, you can download the .zip at https://github.com/phpmyadmin/phpmyadmin/archive/master.zip)?

**Comment by cFreed-AKA-Alf** [comment]
Hi ibennetch, thanks for the answer.

And thanks for what you're proposing, but I'm sorry: I'll not try anything now because, as I already said, I didn't experience the issue since I could escape from it, so:

 - I currently don't _need_ anything about that
 - more over, due to the way it happened first, persisted a while, then disappeared and didn't come back whatever I tried later (notably again launch PMA 4.6.0 when a PMA 4.1.4 instance is already open in another tab) I'm almost sure I cannot reproduce the issue
 - in addition (even if somewhat contradictory!), I prefer not to temp fate for now, because it might happen that it come back and I can't escape as easy as the previous time :)

But, again, thanks for the reactivity!

**Comment by ibennetch** [comment]
Thanks for the reply. Given that it's difficult to reproduce, and that it looks like this part of the code has been improved, I think we can close this for now. Thanks.

**Comment by cFreed-AKA-Alf** [comment]
Despite what I wrote previously, I wanted to at least have an idea of what had been changed in the `999db14` version you cited.
Then I'd a glance to the code at the involved part of `libraries/Config.php`, and I notice that the call to the not-existing `PMA_langset()` function was replaced by a series of statements before the (previously next) cookie setting.

So it enforces the assumption that the issue is now fixed! Thanks.
